src/constants.jl

```diff
@@ -0,0 +1,125 @@
+
+"""
+Base URL for TIGER/Line data with two parameters for year and layer.
+"""
+BASE_TIGER_URL = FormatExpr("https://www2.census.gov/geo/tiger/TIGER{}/{}/")
+
+# "https://www2.census.gov/geo/tiger/GENZ2018/shp/cb_2018_us_state_5m.zip"
+
+"""
+A dictionary mapping human-readable keys to TIGER/Line dataset codes and their associated descriptions.
+
+> **Source:** `https://www2.census.gov/geo/tiger/TIGER2017/2017_TL_Shapefiles_File_Name_Definitions.pdf`
+
+## Keys
+
+- `"address_range_rel"` (**ADDR**) - Address Range Relationship File
```

Comment:
what are the three different things here?  julia name, tiger name, description?  We should describe this explicitly